back/app/services/input_ui_schema_generator_service.rb

```diff
@@ -44,7 +46,7 @@ def category_for(fields, category_id, translation_key)
 
     {
       type: 'Category',
-      label: I18n.t(translation_key),
+      label: (I18n.t(translation_key) if translation_key),
```

Comment:
As the `translation_key` parameter can be either a string or nil, I wonder if it wouldn't be more appropriate to make it an optional paramter?